### ISSUE TITLE
Error: Call to undefined method GuzzleHttp\Handler\MockHandler::reset()

### ISSUE THREAD

**Issue by andrew-beak-property-logic** [guzzle/guzzle#2573]
**Guzzle version(s) affected**: ^6.4
**PHP version**: 7.2.26
**cURL version**: 7.64.0

**Description**  
The [manual](http://docs.guzzlephp.org/en/stable/testing.html) says that to test a client you can do this:

```
// Reset the queue and queue up a new response
$mock->reset();
$mock->append(new Response(201));
```

However, there is no such method defined on the MockHandler class and you get the error:

`Error: Call to undefined method GuzzleHttp\Handler\MockHandler::reset()`

**How to reproduce**  

```
   public function setUp()
    {
        parent::setUp();

        // set up Guzzle mocking
        $this->mock = new MockHandler([]);
        $handlerStack = HandlerStack::create($this->mock);
        $guzzleClient = new Client(['handler' => $handlerStack]);
       // instantiate object under test
        $repository = new MyRepository();
        $this->service = new MyService($repository, $guzzleClient);
    }

    public function testController()
    {
        $this->mock->reset();
    }
```

**Possible Solution**  

**Additional context**

**Comment by alexeyshockov** [comment]
@andrew-beak-property-logic, could you please check `MockHandler.php` file in your `vendor` directory? Try to reinstall all the vendors?..

`::reset()` method is here as far as I can see (https://github.com/guzzle/guzzle/blob/6.5/src/Handler/MockHandler.php#L178-L181).

**Comment by andrew-beak-property-logic** [comment]
Ah, I was using 6.4.1, but I see that it's fixed in 6.5.2

> composer show -l | grep -i guzzle
> guzzlehttp/guzzle                     6.4.1    ! 6.5.2    Guzzle is a PHP HTTP client library
> guzzlehttp/promises                   v1.3.1   = v1.3.1   Guzzle promises library
> guzzlehttp/psr7                       1.6.1    = 1.6.1    PSR-7 message implementation that also provides common utility methods
> guzzlehttp/ringphp                    1.1.1    = 1.1.1    Provides a simple API and specification that abstracts away the details of HTTP into a single PHP function.
> Package guzzlehttp/ringphp is abandoned, you should avoid using it. No replacement was suggested.
> guzzlehttp/streams                    3.0.0    = 3.0.0    Provides a simple abstraction over streams of data
> Package guzzlehttp/streams is abandoned, you should avoid using it. No replacement was suggested.
>
